contributing.md

```diff
@@ -0,0 +1,79 @@
+
+
+## Adding reader tests
+
+Reader tests come in two flavors:
+
+ 1. Generated tables, which are specified from PySpark code in `dat/generated_tables.py`.
+ 2. External tables, which are checked in directly at `external-tables`.
+
+Both tables types have their full test data checked into the `out/reader_tests/` folder.
+
+## Writer tests
+
+TBD
+
+
+## Installation
+
+You can install the dependencies for this project by running `poetry install`.  Run `poetry shell` to activate the virtual environment.    If `poetry shell` does not activate the environment on your machine, you may need to run this command:
+
+```
+source "$( poetry env list --full-path | grep Activated | cut -d' ' -f1 )/bin/activate"
+```
+
+Run `python -m dat.main` to make sure everything is working properly.  You should see instructions on how to use DAT, including the commands you can run.
+
+## How to build tables
+
+Create the reference tables with `python -m dat.main write-generated-reference-tables`.
+
+This will create reference tables as follows:
+
+```
+out/
+  tables/
+    generated/
+      reference_table_1/
+        delta/
+          _delta_log/
+            000.json
+            001.json
+          parquet_files
+        parquet/
+          table_content.parquet
```

Comment:
this is out of date. there's no `latest`, `v1` etc folders.

also there's no `case_metadata.json`